specs/git/rebase_master_into_current_branch.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: Set master as upstream and rebase master into feature branch
+command: |-
+    git branch -u master
+    git checkout master
+    git pull
+    git checkout {{featurebranch}}
```

Comment:
Instead of having the user input the feature branch as an argument, you could use `git rev-parse --abbrev-ref HEAD` to get the feature branch - or use `git checkout -`. 

I think this is far more ergonomic without the argument!

Reply:
Great idea, will update to `git checkout -`